lib/pmt/prompt.go

```diff
@@ -0,0 +1,581 @@
+package pmt
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+	"regexp/syntax"
+	"strconv"
+	"strings"
+
+	"github.com/chef/automate/lib/io/fileutils"
+	"github.com/manifoldco/promptui"
+)
+
+const (
+	FAIL_MSG         = "prompt failed %v"
+	INVALID_INT      = "invalid int"
+	PARSE_INT_FAIL   = "parse int failed %v"
+	REGEX_CHECK_FAIL = "regex check failed (%v) "
+)
+const esc = "\033["
+const moveDown = esc + "1B"
+const moveUp = esc + "1A"
```

Comment:
These constants are defined at more than one place, can we have common ones instead of defining at multiple places?

Reply:
different packages, don't want to import them as they mainly are needed to test prompt functions.